kingpin/bin/deploy.py

```diff
@@ -93,8 +93,8 @@ def main():
                        options={'macro': options.json,
                                 'tokens': env_tokens})
         yield runner.execute()
-    except actor_exceptions.ActorException:
-        log.error('Kingpin encountered mistakes during the play.')
+    except actor_exceptions.ActorException as e:
+        log.error('Kingpin encountered mistakes during the play: %s' % e)
```

Comment:
We're gonna have a merge-agreement: https://github.com/Nextdoor/kingpin/pull/169/files#diff-fb0822d0582573214d6359739e769803R98